### PR TITLE
Add `byteSize.perSecond()`

### PR DESCRIPTION
**Pull request by kasperisager**

### CODE REVIEW

**Comment by kasperisager** [comment]
@mafintosh This is ready for review.

File: index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module.exports = function tinyByteSize (bytes, space = false) {
+const byteSize = module.exports = function tinyByteSize (bytes, space = false) {
```
Comment (mafintosh) [inline index.js]: module.exports = exports = function byteSize...

File: index.js [hunk 2]
```diff
@@ -11,6 +11,10 @@ module.exports = function tinyByteSize (bytes, space = false) {
   return fmt(bytes / 1e24, 'YB', space)
 }
 
+module.exports.perSecond = function perSecond (bytes, ms = 1000, space = false) {
```
Comment (mafintosh) [inline index.js]: exports.perSend = function perSecond

then esm works :) (might work with o/ but i don't think so)

File: package.json [hunk 3]
```diff
@@ -3,8 +3,14 @@
...
   "main": "index.js",
+  "scripts": {
+    "test": "standard && brittle test.mjs"
+  },
   "dependencies": {},
-  "devDependencies": {},
+  "devDependencies": {
+    "brittle": "^3.0.4",
+    "standard": "^17.0.0"
+  },
```
Comment (mafintosh) [inline package.json:16]: add that files thing also, that excludes the tests, i'm a big fan of that now! :D

**Comment by mafintosh** [comment]
1.1.0

**Pull request merged**